src/transformers/utils/auto_docstring.py

```
@@ -1411,8 +1411,8 @@ def _process_regular_parameters(
                     param_type = f"[`{class_name}`]"
                 else:
                     param_type = f"[`{param_type.split('.')[-1]}`]"
-            elif param_type == "" and False:  # TODO: Enforce typing for all parameters
-                print(f"🚨 {param_name} for {func.__qualname__} in file {func.__code__.co_filename} has no type")
+            # elif param_type == "" and False:  # TODO: Enforce typing for all parameters
+            #     print(f"🚨 {param_name} for {func.__qualname__} in file {func.__code__.co_filename} has no type")
```

cc @yonigozlan to make sure it's a valid change

---

Thanks @qubvel, @cyyever any reason why you removed this? This is a soft safeguard for auto_docstring, which will be changed to a hard one one soon (it will raise an error instead of printing)

---

Not sure what happened here, must have left this when debugging. We can leave it like that and I'll check in another PR if we should enable this back and make it a hard requirement (raise an error)